pages/_app.tsx

```diff
@@ -1,18 +1,21 @@
-import type { AppProps } from "next/app";
-import LayoutComponent from "@/components/Layout/LayoutComponent";
-import { Provider } from "react-redux";
-import { wrapper } from "@/src/store";
-import "@/styles/globals.css";
+import { ChakraProvider } from '@chakra-ui/react';
```

Comment:
format this imports as per our convention

Reply:
done